src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -190,6 +194,10 @@ describe('ReactElement', function() {
     expect(React.isValidElement('string')).toEqual(false);
     expect(React.isValidElement(React.DOM.div)).toEqual(false);
     expect(React.isValidElement(Component)).toEqual(false);
+    expect(React.isValidElement({ type: 'div', props: {} })).toEqual(false);
+
+    var jsonElement = JSON.stringify(React.createElement('div'));
+    expect(React.isValidElement(JSON.parse(jsonElement))).toBe(true);
```

Comment:
Ok, just so I know, all of these tests here are explicitly for an environment where we don't have a native `Symbol`. In that case we expect `React.isValidElement(JSON.parse(JSON.stringify(element)))` to be `true`.

*BUT* if we do have a native `Symbol` (and weren't deleting it properly), then we would not expect this test to pass?